README.md

```diff
@@ -28,7 +28,7 @@ this, see the [XNATpy documentation](https://xnat.readthedocs.io/en/latest/stati
 By default, output of the tool is in turtle format at stdout to make for easy piping, but it can be
 written in a variety of formats to a file, too. For all options, see `img2catalog --help`:
 
-```text
+```sh
```

Comment:
```suggestion
```txt
```

Reply:
Dit moet toch text zijn, want het is geen shell script.

Reply:
Ben ik niet mee eens. Nu heb je geen syntax highlighting. Ziet er veel beter uit wanneer je shell scripot er van maakt ;-) Ook al is het sh script. sh is niets meer dan terminal commands achter elkaar

Reply:
Alleen de tekst hier is geen commando maar documentatie....